.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - "iojs"
+  - 6
+  - 5
+  - 4
+  - 0.12
+  - 0.11
+  - 0.10
```

Comment:
You stripped away the quotes, please keep them for reducing the noise in this PR =/